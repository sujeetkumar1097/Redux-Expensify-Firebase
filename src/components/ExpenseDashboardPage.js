import React from 'react';
import ExpenseList from './ExpenseList'
import ExpenseListFilter from './ExpenseListFilter'
import ExpenseSummary from './ExpenseSummary'

const ExpenseDashboardPage = () => (
    <div>
        <ExpenseSummary/>
        <ExpenseListFilter/>
       <ExpenseList/>
    </div>
    
)

export default ExpenseDashboardPage