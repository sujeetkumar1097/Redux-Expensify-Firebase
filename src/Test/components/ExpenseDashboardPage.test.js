import React from 'react'
import {shallow} from 'enzyme'
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage'

test('Not Found Page Render Correctly', () =>{
    const wrapper = shallow(<ExpenseDashboardPage/>)
    expect(wrapper).toMatchSnapshot()
})