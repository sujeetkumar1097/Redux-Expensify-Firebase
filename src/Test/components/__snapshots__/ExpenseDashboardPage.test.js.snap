// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Page Render Correctly 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
</div>
`;
